en/02_Developer_Guides/06_Testing/index.md

```diff
@@ -15,7 +15,7 @@ process of creating and managing tests.
 If you're more familiar with unit testing, but want a refresher of some of the concepts and terminology, you can browse
 the [Testing Glossary](testing_glossary). To get started now, follow the installation instructions below.
 
-You should also read over the [PHPUnit manual](https://docs.phpunit.de/en/9.6/). It provides a lot of
+You should also read over the [PHPUnit manual](https://docs.phpunit.de/en/11.4/). It provides a lot of
```

Comment:
I didn't review the main content of this page - but seemed sensible to update these links while I was in here doing the flush changes.